primitives/src/lib.rs

```diff
@@ -413,3 +413,94 @@ pub mod staking {
         };
     }
 }
+
+pub mod crypto {
+    use core::marker::PhantomData;
+
+    use parity_scale_codec::{Decode, Encode};
+    use scale_info::TypeInfo;
+    use sp_runtime::RuntimeAppPublic;
+    use sp_std::vec::Vec;
+
+    use super::AuthoritySignature;
+
+    pub type NodeIndex = u64;
+    pub type NodeCount = u64;
```

Comment:
Hm, perhaps it's better to just use plain `u64`s here (for both), the type names are similar to the actually different types from ABFT and that might be confusing to the reader. We don't get much of of the type names anyway, since they are used in a pretty limited context.

Reply:
Removed.